.env.example

```diff
@@ -9,3 +8,13 @@ GITHUB_AUTH_ENDPOINT=https://github.com/login/oauth/authorize
 GITHUB_CALLBACK_URL=http://localhost:3000/auth/github
 GITHUB_TOKEN_ENDPOINT=https://github.com/login/oauth/access_token
 GITHUB_USERINFO_ENDPOINT=https://api.github.com/user/emails
+
+AI_PROVIDER=openai|anthropic (default: openai)
+EMBEDDING_MODEL=(default: text-embedding-ada-002)
+DISTANCE_METRIC=euclidean|cosine|dotProduct (default: cosine)
+VECTOR_DIMENSIONS=(default: 1536)
+ENTITY_VECTOR_PATH=(default: embedding)
+
+OPENAI_API_KEY=
+ANTHROPIC_API_KEY=
+VOYAGE_API_KEY=
```

Comment:
LLM_NAME= is missing in .env.example

Reply:
In code you use NODB_VECTOR_INDEX which does not exist here as well.
ENTITY_VECTOR_PATH should be renamed to NODB_VECTOR_PATH here.